csrc/logical_domain_map.cpp

```diff
@@ -214,14 +214,16 @@ std::unordered_map<IterDomain*, IterDomain*> PairwiseLogicalDomainMap::map(
       NVF_ERROR(false, "Producer did not match any LinearOp input.")
     }
 
+    bool k_bcast = op->inA()->as<TensorView>()->axis(-1)->isBroadcast();
```

Comment:
naive question, do we not allow broadcast k between `inA` and `inB`? Hence here we only need to check for one but not the other.

Reply:
We should not allow that. I'm not sure if it's checked yet in the op.

Reply:
I'll add a check, thanks for catching it @jjsjann123. I erroneously recalled already having that check.